requests_futures/sessions.py

```diff
@@ -109,3 +109,73 @@ def close(self):
         if self._owned_executor:
             self.executor.shutdown()
 
+    def get(self, url, **kwargs):
+        r"""
```

Comment:
See these have just been copied over as-is, but still wondering why they're marked as `r'` for regexes.